src/main/java/org/spongepowered/api/command/args/CommandFlags.java

```diff
@@ -336,7 +340,7 @@ protected Object parseValue(CommandSource source, CommandArgs args) throws Argum
 
         Builder() {}
 
-        private static final Function<String, CommandElement> MARK_TRUE_FUNC = input -> markTrue(Text.of(input));
+        private static final Function<String, CommandElement> MARK_TRUE_FUNC = input -> new FlagElement(Text.of(input), null);
```

Comment:
The name of this should change - it no longer marks as true!

Reply:
`FlagElement` will still add `true` to the context so it's backwards compatible. This is only used once now so it might be best to just inline it anyways.

Reply:
It's used multiple times, just in the same function. I think we're just better off keeping it in a field, as I say, it just needs renaming because [it doesn't necessarily mark as true](https://github.com/SpongePowered/SpongeAPI/pull/2033/files#diff-bddcca313ee050f07039ec6b24e4653eR508).

Reply:
`valueElement` at that point is always `null`. I don't see where other uses of `MARK_TRUE_FUNC` are, I can only find one in [`#flag(String...)`](https://github.com/SpongePowered/SpongeAPI/pull/2033/files#diff-bddcca313ee050f07039ec6b24e4653eR391).

Reply:
Oh, yeah... I'm tired. You're right. Bin it then.